### ISSUE TITLE
Support Arma 3 HTML preset files

### ISSUE THREAD

**Issue by killbox998** [muttleyxd/arma3-unix-launcher#56]
Would I be possible to load a mod list using a file from the normal Arma 3 launcher, which uses HTML files.

**Comment by muttleyxd** [comment]
I think you (or someone else) asked me about this on Discord, yet I've forgot about it.

Could you provide me an example HTML file?

**Comment by killbox998** [comment]
[Arma_3_Preset_ACE_Vanilla.html.txt](https://github.com/muttleyxd/arma3-unix-launcher/files/4187240/Arma_3_Preset_ACE_Vanilla.html.txt)

**Comment by TriMoon** [comment]
Here is a bash script to aid you in your task :wink:

- [filter.sh.txt](https://github.com/muttleyxd/arma3-unix-launcher/files/4229767/filter.sh.txt) \ _Remove the <kbd>.txt</kbd> extension and give it <kbd>+x</kbd> permissions..._ \
Example usage: `./filter.sh Arma_3_Preset_ACE_Vanilla.html`
   - The real filtering done can also be used as a separate file to be used by sed's <kbd>--file=</kbd> argument, but then you need to parse the output a bit your self as it does not give the same clean output as the bash script will provide: [script.sed.txt](https://github.com/muttleyxd/arma3-unix-launcher/files/4229820/script.sed.txt)

When you run with the above file from @killbox998 you will get:
```console
http://steamcommunity.com/sharedfiles/filedetails/?id=450814997        @CBA_A3
http://steamcommunity.com/sharedfiles/filedetails/?id=333310405        @Enhanced Movement
http://steamcommunity.com/sharedfiles/filedetails/?id=861133494        @JSRS SOUNDMOD
http://steamcommunity.com/sharedfiles/filedetails/?id=567737932        @Reduced Weapon Sway
http://steamcommunity.com/sharedfiles/filedetails/?id=463939057        @ace
http://steamcommunity.com/sharedfiles/filedetails/?id=1523363834       @HAFM NAVY Core &amp; Weapons part
http://steamcommunity.com/sharedfiles/filedetails/?id=498740884        @ShackTac User Interface
http://steamcommunity.com/sharedfiles/filedetails/?id=620019431        @task_force_radio
http://steamcommunity.com/sharedfiles/filedetails/?id=770418577        @Splendid Smoke
http://steamcommunity.com/sharedfiles/filedetails/?id=713709341        @Advanced Rappelling
http://steamcommunity.com/sharedfiles/filedetails/?id=730310357        @Advanced Urban Rappelling
http://steamcommunity.com/sharedfiles/filedetails/?id=766491311        @ACE 3 Extension (Animations and Actions)
http://steamcommunity.com/sharedfiles/filedetails/?id=723217262        @Achilles
http://steamcommunity.com/sharedfiles/filedetails/?id=708250744        @ACEX
http://steamcommunity.com/sharedfiles/filedetails/?id=623475643        @3den Enhanced
http://steamcommunity.com/sharedfiles/filedetails/?id=825174634        @Suppress
http://steamcommunity.com/sharedfiles/filedetails/?id=330941960        @USS Iowa BB-61
http://steamcommunity.com/sharedfiles/filedetails/?id=932001853        @NATO Corvette [v1.1.1]
http://steamcommunity.com/sharedfiles/filedetails/?id=1362114638       @Hellenic Armed Forces MOD - Submarines
http://steamcommunity.com/sharedfiles/filedetails/?id=1118982882       @Hellenic Armed Forces Mod (HAFM) - NAVY
```

**Comment by muttleyxd** [comment]
@killbox998 @TriMoon 
Fixed by 27a9661.

Feel free to reopen if issue persists.
